### PR TITLE
all: fix docstring names

### PR DESCRIPTION
**Pull request by zoereco**
Found comments that do not start with exported element name and fixed (https://tip.golang.org/doc/comment).

### CODE REVIEW

File: crypto/blake2b/blake2x.go [hunk 1]
```diff
@@ -13,11 +13,11 @@ import (
 // XOF defines the interface to hash functions that
 // support arbitrary-length output.
 type XOF interface {
-	// Write absorbs more data into the hash's state. It panics if called
+	// Writer absorbs more data into the hash's state. It panics if called
```
Comment (holiman) [inline crypto/blake2b/blake2x.go:16]: The comment here referred to the method that the `io.Writer` interface brings to this composite interface, namely:
```golang
type Writer interface {
	Write(p []byte) (n int, err error)
}
```
So please revert this change and the one below re `io.Reader`

File: log/handler_glog.go [hunk 2]
```diff
@@ -192,7 +192,7 @@ func (h *GlogHandler) Handle(_ context.Context, r slog.Record) error {
 		frame, _ := fs.Next()
 
 		for _, rule := range h.patterns {
-			if rule.pattern.MatchString(fmt.Sprintf("%+s", frame.File)) {
+			if rule.pattern.MatchString(fmt.Sprintf("%s", frame.File)) {
```
Comment (holiman) [inline log/handler_glog.go:195]: Why did this change?

File: p2p/discover/v5wire/msg.go [hunk 3]
```diff
@@ -61,7 +61,7 @@ type (
 		Nonce Nonce
 	}
 
-	// WHOAREYOU contains the handshake challenge.
+	// Whoareyou contains the handshake challenge.
```
Comment (holiman) [inline p2p/discover/v5wire/msg.go:64]: I think this is deliberate, and @fjl wants it like this. `WHOAREYOU` is a protocol message. How come you only changed the `WHOAREYOU`, but not `PING` on L78 and `PING` on L84 etc? They should all use the same format

**Review by holiman — CHANGES_REQUESTED** [review]
Thanks!

File: p2p/discover/v5wire/msg.go (hunk 3 above)
Comment (holiman) [reply p2p/discover/v5wire/msg.go:64]: >  But it's not super important to me to keep it that way especially if it is inconsistent.

Well, as of yet it's kind of consistent, but this PR makes it inconsistent (since PING/PONG) is untouched

File: log/handler_glog.go (hunk 2 above)
Comment (holiman) [reply log/handler_glog.go:195]: Well, you are matching a string against some regexp. Why change it if there's no need for it? 
Please revert

Comment (holiman) [inline log/handler_glog.go:195]:
```suggestion
			if rule.pattern.MatchString(fmt.Sprintf("+%s", frame.File)) {
```

**Review by holiman — APPROVED** [review]
LGTM

**Comment by engr0001** [comment]
amazing experience.its a great project.....
